securityscanutils/securityscan.go

```diff
@@ -236,15 +241,21 @@ func (r *SecurityScanRepo) RunMarkdownScan(ctx context.Context, release *github.
 		output := path.Join(outputDir, fileName)
 		_, vulnFound, err := r.trivyScanner.ScanImage(ctx, imageWithRepo, markdownTplFile, output)
 		if err != nil {
-			return eris.Wrapf(err, "error running image scan on image %s", imageWithRepo)
+			if !errors.Is(err, ImageNotFoundError) {
+				return eris.Wrapf(err, "error running image scan on image %s", imageWithRepo)
+			}
+			vulnerabilityMd += fmt.Sprintf("# %s\n\n %s", imageWithRepo, ImageNotFoundError)
 		}
 
 		if vulnFound {
 			trivyScanMd, err := ioutil.ReadFile(output)
 			if err != nil {
 				return eris.Wrapf(err, "error reading trivy markdown scan file %s to generate github issue", output)
 			}
-			vulnerabilityMd += fmt.Sprintf("# %s\n\n %s\n\n", imageWithRepo, trivyScanMd)
+			if r.Opts.DeveloperDebugInstructions != "" {
+				vulnerabilityMd += fmt.Sprintf("%s\n\n", r.Opts.DeveloperDebugInstructions)
```

Comment:
Do you mind posting a screenshot of example usage here?

Reply:
Added to the PR description